latest/src/app/documentation/demos/full-page-layouts/forms-layout/forms-layout.ts

```diff
@@ -0,0 +1,143 @@
+        <input clrInput type="text" class="clr-col-12 clr-col-sm-4 clr-col-lg-6"/>
+    </clr-input-container>
+    <clr-input-container class="clr-col-12 clr-col-lg-6">
+        <label class="clr-col-12 clr-col-sm-2 clr-col-lg-3">Phone</label>
+        <input clrInput type="text" class="clr-col-12 clr-col-sm-4 clr-col-lg-6"/>
+    </clr-input-container>
+
+    <h4 class="clr-col-12">User Data</h4>
+    <clr-input-container class="clr-col-12 clr-col-lg-6">
+        <label class="clr-col-12 clr-col-sm-2 clr-col-lg-3">Username</label>
+        <input clrInput type="text" class="clr-col-12 clr-col-sm-4 clr-col-lg-6"/>
+    </clr-input-container>
+    <clr-password-container class="clr-col-12 clr-col-lg-6">
+        <label class="clr-col-12 clr-col-sm-2 clr-col-lg-3">Password</label>
+        <input clrPassword type="password" class="clr-col-12 clr-col-sm-4 clr-col-lg-6"/>
+    </clr-password-container>
+
+    <h4 class="clr-col-12">Other Data</h4>
+    <clr-date-time-container class="clr-col-12 clr-col-lg-6 clr-row">
+        <clr-date-container class="clr-col-12">
+            <label class="clr-col-12 clr-col-sm-2 clr-col-lg-3">Start date & time</label>
+            <input clrDate type="text" class="clr-col-12 clr-col-sm-10 clr-col-lg-9"/>
+        </clr-date-container>
+        <input clrTime type="time" />
+    </clr-date-time-container>
+    <clr-input-container class="clr-col-12 clr-col-lg-6">
+        <label class="clr-col-12 clr-col-sm-2 clr-col-lg-3">End time</label>
+        <input clrInput type="time" class="clr-col-12 clr-col-sm-3 clr-col-lg-5"/>
+    </clr-input-container>
+    <clr-input-container class="clr-col-12 clr-col-lg-6">
+        <label class="clr-col-12 clr-col-sm-2 clr-col-lg-3">Daily Donation</label>
+        <input clrInput clrNumeric [clrUnit]="'€'" type="text" class="clr-col-12 clr-col-sm-3 clr-col-lg-5"/>
+    </clr-input-container>
+    <clr-checkbox-container class="clr-col-12 clr-col-lg-6">
+        <label class="clr-col-12 clr-col-sm-2 clr-col-lg-3">Newsletter Subscriptions</label>
+        <clr-checkbox-wrapper>
+            <label>Weekly</label>
+            <input clrCheckbox class="clr-col-12 clr-col-sm-6 clr-col-lg-6" type="checkbox"/>
+        </clr-checkbox-wrapper>
+        <clr-checkbox-wrapper>
+            <label>Monthly</label>
+            <input clrCheckbox class="clr-col-12 clr-col-sm-6 clr-col-lg-6" type="checkbox"/>
+        </clr-checkbox-wrapper>
+    </clr-checkbox-container>
+    <clr-textarea-container class="clr-col-12 clr-col-lg-6">
+        <label class="clr-col-12 clr-col-sm-2 clr-col-lg-3">Comment / Feedback</label>
+        <textarea clrTextarea type="text" class="clr-col-12 clr-col-sm-10 clr-col-lg-9"></textarea>
+    </clr-textarea-container>
+</form>
+`;
+
+@Component({
+    selector: "clr-forms-layout-demo",
+    templateUrl: "./forms-layout.demo.html"
+})
+export class FormsLayout implements OnInit {
+    codeExample = CODE_EXAMPLE;
+
+    withCommandBar = false;
```

Comment:
can be removed here, will always be false in this route (also remove html for this)